packages/cli/src/services/project-parser.ts

```diff
@@ -72,7 +72,7 @@ async function loadAllCheckFiles (
   for (const checkFile of checkFiles) {
     // setting the checkFilePath is used for filtering by file name on the command line
     Session.checkFileAbsolutePath = checkFile
-    Session.checkFilePath = path.relative(directory, checkFile)
+    Session.checkFilePath = pathToPosix(path.relative(directory, checkFile))
```

Comment:
Do we need `pathToPosix` here (and on line 75 above)? Do we have a bug or is this just to refactor the code?

I'm wondering if in some places we should just keep the platform dependent path. In the case of `checkFilePath` I think it's mainly used for filtering with `npx checkly  test <some-path-filter>`. Maybe it's more intuitive for Windows users to be able to still use the Windows path separator, though: `npx checkly test \path\to\my_test.spec.js`.

Reply:
@clample , I convert all paths because I read the docs for `glob` and they suggest to always use the Unix path separator, even on Windows:
![image](https://user-images.githubusercontent.com/5315705/223098838-00b25cdc-2799-45d2-823e-303f95c5539d.png)

Reply:
@nahuelon interesting catch 🤔 

 I think that `npx checkly  test <some-path-filter>` doesn't actually go through the `glob` package. Instead, it uses a regex (code snippets below). It might still make sense to switch to `/`, though, like you mention.
 
 Either way, if we don't really need this to fix https://github.com/checkly/checkly-cli/issues/568, I propose just moving the `pathToPosix` change out of this PR. It would be easier to discuss what the behavior should be with a separate Slack thread or PR, and we could just merge the `.page.js` fix right away.
 
https://github.com/checkly/checkly-cli/blob/60a3464d0d73566b464249377c231adf099dedbd/packages/cli/src/commands/test.ts#L126-L128

https://github.com/checkly/checkly-cli/blob/60a3464d0d73566b464249377c231adf099dedbd/packages/cli/src/services/test-filters.ts#L6-L10